packages/superset-ui-control-utils/src/shared-controls.tsx

```diff
@@ -111,8 +111,8 @@ const groupByControl: ControlConfig = {
   default: [],
   includeTime: false,
   description: t('One or many controls to group by'),
-  optionRenderer: (c: ColumnMeta) => <ColumnOption showType column={c} />,
-  valueRenderer: (c: ColumnMeta) => <ColumnOption column={c} />,
+  optionRenderer: c => <ColumnOption showType column={c} />,
+  valueRenderer: c => <ColumnOption column={c} />,
   valueKey: 'column_name',
   allowAll: true,
   filterOption: (opt: ColumnMeta, text: string) =>
```

Comment:
@ktmud It seems that this code hasn't been updated with the changes from [this commit](https://github.com/preset-io/incubator-superset/commit/81ab8dd8b46f7291171e674cd48d498c1efde409). We can update that as a fast-follow to this PR, unless you have the wherewithall to update (and properly type) it in this PR.

Reply:
I'll slip in.